### PR TITLE
Fix install prefix

### PR DESCRIPTION
**Pull request by isuruf**
Fixes https://github.com/conda-forge/llvmdev-feedstock/issues/244

### CODE REVIEW

File: recipe/build.sh [hunk 1]
```diff
@@ -73,3 +75,11 @@ cmake --install .
...
 # Clean up after build
 rm -rf "${PREFIX}/lib/libc++.tbd"
+
+if [[ "$target_platform" == "$build_platform" ]]; then
+  RESOURCE_DIR=$(${PREFIX}/bin/clang -print-resource-dir)
+  if [[ "${RESOURCE_DIR}" != "${INSTALL_PREFIX}" ]]; then
+    echo "Wrong install prefix (${INSTALL_PREFIX}). Should match ${RESOURCE_DIR}"
+    exit 1
+  fi
+fi
```
Comment (h-vetinari) [inline recipe/build.sh:85]: Would it make sense to add such a test on windows as well?

**Review by h-vetinari — APPROVED** [review]
Thanks for fixing this!

File: recipe/build.sh (hunk 1 above)
Comment (isuruf) [reply recipe/build.sh:85]: Too lazy to write it in batch script. PRs welcome.

**Comment by isuruf** [comment]
We also need to fix https://github.com/conda-forge/clang-win-activation-feedstock/blob/main/recipe/activate-clang_win-64.bat#L13

**Pull request merged**